### DISCUSSION TITLE
Scaling of Strokes

### DISCUSSION THREAD

**Discussion by avantBentlyNevada** [dotnetprojects/SVGImage#59] [Q&A]
Hey all, I just wanted to verify that how strokes are handled when scaling an image is as designed behavior. Currently when scaling an image the strokes of the image scale with it.  This can be seen in the ClipArtViewer with all of the images that contain visible strokes. However, in a program like Adobe Illustrator this is not the default behavior. In order for the strokes to scale the image Scale Corners, and Scale Strokes and Effects need to be enabled. I don't want to influence the answer, but I am assuming the SVGImage behavior, to scale the complete image, is to protect the integrity of the artist's image.

**Comment by paulushub** [comment]
The scaling is WPF defined feature and not directly implemented by SVGImage. 
After SVGImage converts the SVG to WPF drawing, all manipulations are done by and through WPF.
